### DISCUSSION TITLE
What is the git repo for broker-etc generated by init image?

### DISCUSSION THREAD

**Discussion by Mohanavalli-A** [arkmq-org/arkmq-org-broker-operator#753] [Q&A]
Hi Team,

Can you please provide the git repo path from where the init image generates the files that go inside the broker /etc directory.

https://github.com/artemiscloud/activemq-artemis-broker-init-image/tree/main/script/cfg/broker-etc has not been updated from past few releases.

Thanks!

**Comment by gaohoward** [comment]
Hi, this dir is used for test purpose only. The real contents comes from artemis cli that is called to create a broker instance in [this script](https://github.com/artemiscloud/activemq-artemis-broker-kubernetes-image/blob/main/modules/activemq-artemis-launch/added/launch.sh#L649). Here is [an example](https://github.com/artemiscloud/artemiscloud-examples/tree/main/operator/init/jdbc) of how custom init container can modify the configure files under etc dir.

Going forward it is recommended to use [brokerProperties](https://github.com/artemiscloud/activemq-artemis-operator/blob/main/docs/help/operator.md#configuring-brokerproperties) to configure broker if possible, instead of using init container. In the future init container will go deprecated.
